src/components/ThemeToggle.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+	<div class="custom-control custom-switch toogleTheme">
+		<input type="checkbox" class="custom-control-input" id="goDark" v-model="isDarkMode" @change="toggleTheme">
+		<label class="custom-control-label toggle" for="goDark">
+			<span class="sun" :class="{'toggle-button': !this.isDarkMode}">
+				<img
+					class="switch-icon"
+					:src="IconSun"
+					width="25px"
+					height="25px"
+					alt="light mode"
+				/>
+			</span>
+
+			<span class="moon" :class="{'toggle-button': this.isDarkMode}">
+				<img
+					class="switch-icon"
+					:src="IconMoon"
+					width="25px"
+					height="25px"
+					alt="dark mode"
+				/>
+			</span>
+		</label>
+	</div>
+</template>
+
+<script>
+import IconMoon from '../styles/img/moon.png';
+import IconSun from '../styles/img/sun.png';
+
+export default {
+	mounted() {
+		// get theme from localstorage
+		let theme = localStorage.getItem('theme');
```

Comment:
I think we should not access localStorage directly from UI components. I would move it to store or service

Reply:
The reason put into localstorage is to prevent theme reset.